getting-started/chat-thread.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Chat and thread"
+weight: 10
+---
+
+# Chat and thread
+
+
+### Basics
+
+In our system chat is divided into threads. Threads are standalone parts of chat and can contain events. In every chat multiple users (customers or agents) can participate and every user can have multiple chats at the same time.
+
+New threads within chat are created on server side, algorithm of this is discussed below in *Advanced* section.
```

Comment:
New threads within chat -> New threads within a chat

are created on server side -> are created on the server side (albo: are created server-side)

algorithm of this is discussed below in *Advanced* section. -> napisałabym: see the Advanced section below for the algorithm